### ISSUE TITLE
add eslint and prettier for code structure and formatting

### ISSUE THREAD

**Issue by ryanfavour4** [TaiwoIgbaro1212/xp-portal__revised#4]
add eslint and prettier for code structure and formatting. rules are meant to indicate if the cofe formatting is ugly alone. 
#### do not disable console.log or any extra extension eg. .tsx .ts .js .jsx

@Thi-mee 
@ryanfavour4 
@Joshthefullstack

**Comment by Thi-mee** [comment]
You're gonna have to do a lot of explanation on this

**Comment by ryanfavour4** [comment]
> You're gonna have to do a lot of explanation on this

Yes sire i will. Big boy josh would be done today so hopefully i can kidnap you when you come back
